pkg/http/api.go

```diff
@@ -692,24 +694,41 @@ func (a *api) getStateStoreWithRequestValidation(reqCtx *fasthttp.RequestCtx) (s
 	return state, storeName, nil
 }
 
-// Route:   "workflows/{workflowComponent}/{workflowName}/{instanceId}",
+// Route:   "workflows/{workflowComponent}/{workflowName}/start?instanceID={instanceID}",
```

Comment:
Should this be passed in the body?

Reply:
No, I'm intentionally trying to use the request body exclusively for the raw workflow input without any Dapr interpretation.

Reply:
Ok the same situation was just addressed here: https://github.com/dapr/dapr/pull/6197#discussion_r1170857312

@mukundansundar  and @artursouza recommended to use HTTP headers instead

Reply:
That's interesting. I'd love to get more context on this statement, which I don't understand:
> My concern is that query args are directly available to be cached and used.

Happy to add support for headers. Is supporting both an option? I do like the convenience of query string parameters.

Reply:
@mukundansundar @artursouza What is your viewpoint here?

Reply:
In this case I agree that queryArgs will be better, as this is specific for an instance and optional.